doc/release/1.7.0-notes.rst

```diff
@@ -1,6 +1,6 @@
-=========================
-NumPy 1.7.0 Release Notes
-=========================
+===========================
+ NumPy 1.7.0 Release Notes
+===========================
```

Comment:
Are you sure this doesn't break the rendering?

Reply:
I built the html file and it looks okay in my browser.

Reply:
I'd prefer this to stay as is - every other release note does not have these spaces